docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -63,15 +63,15 @@ If you log in with GitHub, Codacy requires the following [app permissions](https
       <td>Read & Write</td>
       <td>Codacy sets the status of commits according to the result of code analysis.</td>
     </tr>
-    <tr>
-      <td>Administration</td>
-      <td>Read & Write</td>
-      <td><a href="#why-does-codacy-ask-for-permission-to-create-ssh-keys">Codacy creates an SSH key</a> on the repository to allow cloning and integrating with your repository.</td>
-    </tr>
     <tr>
       <td>Contents</td>
       <td>Read-Only</td>
-      <td>Codacy accesses repository contents to provide faster code coverage analysis and as part of an initiative to use GitHub App tokens instead of SSH keys when cloning repositories for code quality analysis.</td>
+      <td>Codacy retrieves repository contents to get installation access tokens, which are used to integrate with your repositories and clone them for code analysis.</td>
+    </tr>
+    <tr>
+      <td>Administration</td>
+      <td>Read & Write</td>
+      <td>This permission will soon be removed and is currently used <strong>exclusively as a fallback mechanism</strong> when the Contents permission is missing. In this case, Codacy <a href="#why-does-codacy-ask-for-permission-to-create-ssh-keys">creates an SSH key on the repository</a> to allow cloning and integrating with your repository.<br/>To avoid disruptions in the future, make sure you <a href="https://docs.github.com/en/apps/using-github-apps/approving-updated-permissions-for-a-github-app">approve Codacy GitHub App updated permissions</a> on your GitHub organization.</td>
```

Comment:
I would suggest the emphasis (strong) in this text should be on _the soon to be deprecated permission_, rather than the current use we have for it.
ex: `<strong>This permission will soon be removed</strong> and is currently used exclusively as a fallback mechanism (...)`

Also, as we are still figuring out possible issues and ironing out details as we go, it's gonna be inherently difficult to align a "release date" for this, so I'd suggest we adapt language to accommodate for this, something like avoiding the term "exclusively": 
> This permission will soon be removed and is currently used when it's not possible to leverage the Repository Contents permission. In this case, Codacy creates an SSH key on the repository to allow cloning and integrating with your repository.
> To future-proof your codacy usage, make sure you that you aprove [approve Codacy GitHub App updated Repository Coontents permission](https://docs.github.com/en/apps/using-github-apps/approving-updated-permissions-for-a-github-app) on your GitHub organization.

It's possible that I'm being creative with the usage of "not possible", I was trying, without lying to encapsulate both the cases that we reverted the behaviour for some testing reason as well as the cases where customers have not accepted the Repository `Contents` permission.

(Also please don't use any of my suggestions verbatim, I'm sure there are better ways to write these things.)

Reply:
> It's possible that I'm being creative with the usage of "not possible", I was trying, without lying to encapsulate both the cases that we reverted the behaviour for some testing reason as well as the cases where customers have not accepted the Repository Contents permission.

I rephrased it according to the current state of the art. 😉